tsconfig.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
We probably don't want to add a tsconfig to the repo without further discussion?

Reply:
It's from a commit labeled TEMP. I guess the original author used it to aid the refactoring.